sdk/include/wdf/kmdf/1.9/wdfassert.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * wdfassert.h
+ *
+ * Windows Driver Framework - Prototypes for Runtime Asserts
+ *
+ * This file is part of the ReactOS WDF package.
+ *
+ * Contributors:
+ *   Created by Benjamin Aerni <admin@bennottelling.com>
+ *
+ * Intended Usecase:
+ *   Kernel mode drivers
+ *
+ * THIS SOFTWARE IS NOT COPYRIGHTED
+ *
+ * This source code is offered for use in the public domain. You may
+ * use, modify or distribute it freely.
+ *
+ * This code is distributed in the hope that it will be useful but
+ * WITHOUT ANY WARRANTY. ALL WARRANTIES, EXPRESS OR IMPLIED ARE HEREBY
+ * DISCLAIMED. This includes but is not limited to warranties of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ */
+#ifndef _WDFASSERT_H_
+#define _WDFASSERT_H_
+
+
+#if (NTDDI_VERSION >= NTDDI_WIN2K)
+
+/* RtlAssert isn't defined in XP and Win2k headers so we do so here */
```

Comment:
Maybe surrounding the definition of this function by
```c
#ifndef _RTLFUNCS_H
...
#endif
```
could be nice too, in case the WDF headers are included along with some of the NDK ones...